docs/docs/sinks/grpc-sink.md

```diff
@@ -65,3 +65,11 @@ Defines the amount of time (in milliseconds) gRPC clients are willing to wait fo
 
 - Example value: `1000`
 - Type: `optional`
+
+### `SINK_GRPC_RESPONSE_RETRY_CEL_EXPRESSION`
```

Comment:
Can you point to the CEL documentation link, and provide some examples for reference with response proto?